packages/buttongroup/src/useButtonGroup.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { useSelection, composeEventHandlers } from '@zendeskgarden/container-selection';
+import closest from 'dom-helpers/query/closest';
+
+export function useButtonGroup(options) {
+  const { selectedItem, focusedItem, getContainerProps, getItemProps } = useSelection(options);
+
+  const getGroupProps = ({ role = 'group', ...other } = {}) => {
+    return {
+      role,
+      ...other
+    };
+  };
+
+  const getButtonProps = ({
+    role = 'button',
+    key,
+    onFocus,
+    selectedAriaKey = 'aria-pressed',
+    ...other
+  } = {}) => {
+    if (typeof key === 'undefined' || key === null) {
```

Comment:
Why are we using the `key` values here? Isn't this a left-over from the old aria-activedescendant model?

Reply:
yep you're right